vru_use_case_scripts/process_duplicates.py

```diff
@@ -97,12 +95,20 @@ def main():
     parser = argparse.ArgumentParser(description='Script to parse ROS Bags into CSV data')
     parser.add_argument('--ros-bag-file', help='ROS Bags File.', type=str, required=True)
     parser.add_argument('--csv-dir', help='Directory to write csv file to.', type=Path, required=True)
+    parser.add_argument('--plot-dir', type=Path)
```

Comment:
Lets just have `args.plots_dir` and have a default instead of `figures_dir` as well.